pkg/exporter/exporter.go

```diff
@@ -77,7 +78,7 @@ func (e *Exporter) Collect(ch chan<- prometheus.Metric) {
 
 // Describe satisfies prometheus.Collector interface
 func (e *Exporter) Describe(ch chan<- *prometheus.Desc) {
-	ch <- e.certExpiry.WithLabelValues("path", "issuer", "alg", "version", "subject", "dns_names", "email_addresses").Desc()
+	ch <- e.certExpiry.WithLabelValues("path", "issuer", "alg", "version", "subject", "dns_names", "email_addresses", "node").Desc()
```

Comment:
Suggest renaming the label to `hostname` since we have plans to add labels for Kubernetes which will overlap.

Reply:
makes sense